src/main/java/frc/robot/RobotContainer.java

```diff
@@ -300,17 +343,22 @@ private void defineButtons() {
   private void configureButtonBindings() {
     if (drivetrainSubsystem != null) {
       resetGyro.whenPressed(() -> drivetrainSubsystem.zeroGyroscope());
+      alignWithHangar.whenPressed(new InstantCommand(() -> hangarAlignOn()))
+        .whenReleased(new InstantCommand(() -> hangarAlignOff()));
     }
 
     if (intakeSubsystem != null && centererSubsystem != null && indexerSubsystem != null) {
-      driverIntakeButton.whenHeld(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem, colorSensorSubsystem, m_rejectSupplier));
-      operatorIntakeButton.whenHeld(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem, colorSensorSubsystem, m_rejectSupplier));
+      driverIntakeButton.whenHeld(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem, colorSensorSubsystem, m_rejectSupplier))
+        .whenPressed(new InstantCommand(() -> intakeDown()))
+        .whenReleased(new InstantCommand(() -> intakeUp()));
```

Comment:
- [ ] Remove this, it adds conflicting calls to setting intake

Reply:
once again, doesn't set intake but it just flips a boolean saying that intake is currently down